build-system/tasks/runtime-test/helpers.js

```diff
@@ -29,7 +29,7 @@ const {green, yellow, cyan, red} = require('ansi-colors');
 const {isTravisBuild} = require('../../common/travis');
 const {Server} = require('karma');
 
-const BATCHSIZE = 4; // Number of Sauce Lab browsers
+const BATCHSIZE = 3; // Number of Sauce Lab browsers
```

Comment:
Can you add a short comment above this line explaining why this is 3, and link to the PR with the results from your batch size exploration?

Reply:
Done.